src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -241,6 +251,17 @@ class Crystal::Doc::Generator
     end
   end
 
+  # adds extra line break to TODO lines
+  def isolate_flag_lines(string)
+    string.split(/\n/).map do |line|
```

Comment:
I was assuming that html passed through the markdown parser unescaped, because that's what I've seen most engines do. Github lets you use certain html tags in markdown. 

I think that it is probably a bad *default* to let html through, but it should at least be an option. However if we went this route we would have to sanitize html to include only certain tags (github has a list they let through their markdown). It might be worth looking at nature markdown implementations to see how this is done.